### PR TITLE
Disable selinux for gardener

### PR DESCRIPTION
**Pull request by 5kt**
**How to categorize this PR?**

/kind fix
/area os
/os garden-linux

**What this PR does / why we need it**:
We need to disable selinux for gardener. 
Fixing the packages-musthave test in order to properly handle excluded packages.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

### CODE REVIEW

**Review by nkraetzschmar — APPROVED** [review]

**Review by gehoern — APPROVED** [review]

File: features/gardener/pkg.exclude [hunk 1]
```diff
@@ -0,0 +1,2 @@
+selinux-basics
+selinux-policy-default
```
Comment (waldiTM) [inline features/gardener/pkg.exclude:2]: Why do you want to exclude those packages?

File: features/server/file.include/etc/kernel/cmdline.d/90-lsm.cfg [hunk 2]
```diff
@@ -0,0 +1 @@
+CMDLINE_LINUX="$CMDLINE_LINUX selinux=1 apparmor=0"
```
Comment (waldiTM) [inline features/server/file.include/etc/kernel/cmdline.d/90-lsm.cfg:1]: Hmm, this does not qualify as "remove", as someone can expect from the commit message.

Comment (5kt) [reply features/server/file.include/etc/kernel/cmdline.d/90-lsm.cfg:1]: True, the wording is unfortunate.

**Comment by 5kt** [comment]
There have been some misunderstandings regarding these changes, closing.

File: features/gardener/pkg.exclude (hunk 1 above)
Comment (5kt) [reply features/gardener/pkg.exclude:2]: They should not be removed, this was a misunderstanding.

**Pull request closed without merging**